### ISSUE TITLE
#5650 Teaser Tiles Text Field

### ISSUE THREAD

**Issue by maxliqd** [liqd/digitalstrategie#348]
**URL:** https://digitalstrategie-dev.liqd.net/ergebinsse/
**user:** Visitor
**expected behaviour:** Tiles look like design
**behaviour:** Text field starts lower than in design, space between image and text field looks smaller too
**important screensize:**
**device & browser:** Mac firefox
**Comment/Question:** 
@mcastro-lqd : There seems to be an ovelap in the designs: square text field is on top of arrow box. Is that intended?
I am not sure about the space between image and text field --> it looks a bit crammed, could also be because of the chosen images, which are not all icons and transparent background like in the designs.
Screenshot?
![grafik](https://user-images.githubusercontent.com/94360651/160109291-4a037681-984f-4df9-9fb7-ba907dd1e2b2.png)

The design:

<img width="869" alt="grafik" src="https://user-images.githubusercontent.com/94360651/160109385-432b9f95-00a5-4597-9264-e694ecbec69d.png">

**Comment by fuzzylogic2000** [comment]
@hmkay So, the problem is more space to the top in the "Digitale Gleichbehandlung..."-Tile? I did not get the rest of it and also do not know what to do to fix this issue.

**Comment by maxliqd** [comment]
@fuzzylogic2000 We discussed this yesterday (@hmkay and @mcastro-lqd and I). The styling is not exactly matching the design (ratio of tile, position of text box), which is why I created the issue when I tested UI/UX. Marcelo thinks Philly probably had good reasons for this and that's why we are putting this issue on hold for a moment, hoping it will go through the approval process anyway. If that goes smoothly, we can delete the issue. Right, @mcastro-lqd?
